test/smoke/src/helpers/appiumHelper.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as cp from "child_process";
+import * as wdio from "webdriverio";
+import * as path from "path";
+import * as mkdirp from "mkdirp";
+let appiumProcess: null | cp.ChildProcess;
+
+export class appiumHelper {
+    // Android UI elements
+    public static RN_RELOAD_BUTTON = "//*[@text='Reload']";
+    public static RN_ENABLE_REMOTE_DEBUGGING_BUTTON = "//*[@text='Debug JS Remotely']";
+    public static RN_STOP_REMOTE_DEBUGGING_BUTTON = "//*[@text='Stop Remote JS Debugging']";
+
+    public static waitTime = 30000;
+    public static runAppium() {
+        const appiumLogFolder = path.join(__dirname, "..", "..", "..", "..", "logArtifacts");
+        mkdirp.sync(appiumLogFolder);
+        const appiumLogPath = path.join(appiumLogFolder, "appium.log");
+        console.log(`*** Executing Appium with logging to ${appiumLogPath}`);
+        let appiumCommand = process.platform === "win32" ? "appium.cmd" : "appium";
+        appiumProcess = cp.spawn(appiumCommand, ["--log", appiumLogPath]);
+    }
+
+    public static terminateAppium() {
+        if (appiumProcess) {
+            console.log(`*** Terminating Appium`);
+            appiumProcess.kill();
+        }
+    }
+
+    public static prepareAttachOptsForAndroidActivity(applicationPackage: string, applicationActivity: string,
+    platformVersion: string = "9", deviceName: string = "emulator-5554") {
```

Comment:
Please avoid magic constants like "9", "emulator-5554". I'd recommend you to create constants class that contains such constants with their descriptions in comments.

Reply:
Agree.